src/data/translations/jp/terms.json

```diff
@@ -1,45 +1,45 @@
 {
 	"available": true,
 	
-	"SortieWinCount": "Sortie Win Count",
-    "SortieLoseCount": "Sortie Lose Count",
-    "SortieWinRate": "Sortie Win Rate",
-    "TotalExpeditions": "Total Expeditions",
-    "ExpeditionSuccessCount": "Expedition Success Count",
-    "ExpeditionSuccessRate": "Expedition Success Rate",
-    "PVPWinCount": "PVP Win Count",
-    "PVPLoseCount": "PVP Lose Count",
-    "PVPWinRate": "PVP Win Rate",
-    "FleetsOwned": "Fleets Owned",
-    "ConstructionDockCount": "Construction Dock Count",
-    "RepairDockCount": "Repair Dock Count",
-    "ShipGirlsOwned": "Ship Girls Owned",
-    "EquipmentItemCount": "Equipment Item Count",
-    "FurnitureOwned": "Furniture Owned",
-    "ServerName": "Server Name",
-    "MaxFleetsAvailable": "Max Fleets Available",
-    "MaxShipGirlsAvailable": "Max Ship Girls Available",
-    "MaxHomeportUpgradesAvailable": "Max Homeport Upgrades Available",
-    "MaxEquipmentAvailable": "Max Equipment Available",
-    "QuestMarkersAvailable": "Quest Markers Available",
-    "ResourceSoftcapLimit": "Resource Softcap Limit",
-    "MainScreenBGM": "Main Screen BGM",
-    "LevelUpNotice": "As your HQ level increases, softcap limit increases, and crafting of new equipment becomes possible.",
-    "MainScreenBGM": "Main Screen BGM",
-    "Experience": "Experience",
-    "Name": "Name",
+	"SortieWinCount": "出撃の勝数",
+    "SortieLoseCount": "出撃の敗数",
+    "SortieWinRate": "出撃の勝率",
+    "TotalExpeditions": "遠征の回数",
+    "ExpeditionSuccessCount": "遠征の成功数",
+    "ExpeditionSuccessRate": "遠征の成功率",
+    "PVPWinCount": "演習の勝数",
+    "PVPLoseCount": "演習の敗数",
+    "PVPWinRate": "演習の勝率",
+    "FleetsOwned": "保有艦隊数",
+    "ConstructionDockCount": "工廠ドック数",
+    "RepairDockCount": "入渠ドック数",
+    "ShipGirlsOwned": "艦娘保有数",
+    "EquipmentItemCount": "装備アイテム保有数",
+    "FurnitureOwned": "家具保有数",
+    "ServerName": "所属",
+    "MaxFleetsAvailable": "最大保有可能 艦隊数",
+    "MaxShipGirlsAvailable": "最大保有可能 艦娘数",
+    "MaxHomeportUpgradesAvailable": "残り母港拡張 可能枠",
+    "MaxEquipmentAvailable": "最大保有可能 装備アイテム数",
+    "QuestMarkersAvailable": "最大受領可能 任務数",
+    "ResourceSoftcapLimit": "最大備蓄可能 各資材量",
+    "MainScreenBGM": "母港執務室BGM",
+    "LevelUpNotice": "艦隊司令部（提督）levelのupによって「最大備蓄可能資材量の増加」「最新の装備アイテムの開発」が可能になります。",

```

Comment:
The hardest part in wording (because its font on ttk's page is so **** small). :dizzy_face: